cvat/apps/lambda_manager/serializers.py

```diff
@@ -5,30 +5,42 @@
 from drf_spectacular.utils import extend_schema_serializer
 from rest_framework import serializers
 
+class AttributeMappingSerializer(serializers.JSONField):
+    def validate(self, attrs):
+        for attr_name in self.initial_data:
+            if not isinstance(self.initial_data[attr_name], str):
+                raise serializers.ValidationError(f"Not valid field value type: {attr_name}, must be string")
```

Comment:
I'm not sure what you're trying to accomplish here. If you just want a string-to-string mapping, then why not use a `DictField`?